lighthouse-core/config/index.js

```diff
@@ -171,16 +172,79 @@ function filterAudits(audits, auditWhitelist) {
   return filteredAudits;
 }
 
-function expandAudits(audits) {
+function expandAudits(audits, paths) {
+  const rootPath = path.join(__dirname, '../../');
+
   return audits.map(audit => {
-    try {
-      return require(`../audits/${audit}`);
-    } catch (requireError) {
+    // Check each path to see if the audit can be located. First match wins.
+    const AuditClass = paths.reduce((definition, auditPath) => {
+      // If the definition has already been found, just propagate it. Otherwise try a search
+      // on the path in this iteration of the loop.
+      if (definition !== null) {
+        return definition;
+      }
+
+      const requirePath = auditPath.startsWith('/') ? auditPath : path.join(rootPath, auditPath);
```

Comment:
I think there's a path.resolve() or something to sort out absolute vs not. Would rather that then us handling the branch.